app/raid.js

```diff
@@ -87,17 +84,18 @@ class Raid {
 		// add some extra raid data to remember
 		raid_data.id = id;
 		raid_data.creation_time = new moment();
-		raid_data.default_end_time = (new moment()).add(settings.default_raid_length, 'milliseconds');
+		raid_data.last_possible_time = new moment(raid_data.creation_time).add(settings.default_raid_length, 'minutes');
```

Comment:
It's been my experience that this scenario doesn't act the way you think it should, which is why I was doing it a really screwed up way.  You'd THINK it would create a new moment object based off the previous "creation time" one, but in fact I THINK it's really just creates a reference.  So I THINK in this scenario creation_time and last_possible_time will be the same now, after you add the time to it.

Can you test this out @Googer?